test/guardian/db_fail_test.exs

```diff
@@ -1,8 +1,7 @@
 defmodule Guardian.DBFailTest do
-  alias Guardian.DB.Token
   use Guardian.DB.TestSupport.CaseTemplate
 
-  test "after_encode_and_sign_in is fails", context do
+  test "after_encode_and_sign_in is fails" do
```

Comment:
This file had a number of warnings which I fixed. I can pull these into a separate PR if it's so desired :+1: